rijkshuisstijl/templates/rijkshuisstijl/components/datagrid/datagrid-row.html

```diff
@@ -39,7 +38,7 @@
 
 
         {% else %}
-            {% if urlize %}
+            {% if urlize and not column.urlize == False %}
```

Comment:
Looks like we can't specify column.urlize = True and not setting urlize?

Reply:
We also need to update the documnetation sections in: https://github.com/maykinmedia/django-rijkshuisstijl/blob/develop/rijkshuisstijl/templatetags/rijkshuisstijl_datagrid.py#L64 and https://github.com/maykinmedia/django-rijkshuisstijl/blob/develop/rijkshuisstijl/templatetags/rijkshuisstijl_datagrid.py#L277